chrome/extension/create_styleurl/CreateStyleURL.js

```diff
@@ -171,6 +172,14 @@ class CreateStyleURLContainer extends React.Component {
           content
         ])
       });
+
+      if (!_.isArray(stylesheets) || stylesheets.length === 0) {
```

Comment:
If you make it setVisible, then you can make this code `this.setVisible(true)` and then `setVisible` would check if its already visible and do nothing